Core/Repositories/FeatureEntry.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Core.Repositories
+{
+    public class FeatureEntry : TableEntity, IFeatureEntry
+    {
+        public bool IsEnabled { get; set; }
```

Comment:
it makes sense adding `FeatureName => RowKey` for the logic incapsulation reasons

Reply:
`+`